addons/diagnostic/xeh/config.cpp

```diff
@@ -2,6 +2,9 @@
 #ifndef EH_DEBUG_ENABLED
     class CfgPatches {
         class cba_diagnostics_xeh_disabled {
+            author = "$STR_CBA_Author";
+            name = "$STR_CBA_Optional_Component";

```

Comment:
It would expand to the above. Not sure about semantics, if you are worried people might be looking for "optional" component then it might be best to leave it like that.